integration_test/third_party_apps_data/applications/sqlserver/metadata.yaml

```diff
@@ -0,0 +1,125 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/sqlserver"
+short_name: SQL Server
+long_name: Microsoft SQL Server
+description: |-
+  The Microsoft SQL Server integration collects metrics from your SQL Server
+  instances and databases. Page, batch, lock, and user connection count information
+  is provided at the instance level. Transaction and transaction log information
+  is provided at the database level. 
+
+  For more information on SQL Server, see the
+  [SQL Server](https://docs.microsoft.com/en-us/sql/sql-server)
+  documentation.
+minimum_supported_agent_version:
+  metrics: 2.10.0
```

Comment:
This needs to be updated, doesn't it?

Reply:
This should be 2.15.0 (2.14.0 is likely going out before this is merged).

Reply:
updated